lib/beaker/host_prebuilt_steps.rb

```diff
@@ -373,6 +380,8 @@ def enable_root_login host, opts
           host.exec(Command.new("sudo -E /sbin/service sshd reload"), {:pty => true})
         elsif host['platform'] =~ /freebsd/
           host.exec(Command.new("sudo /etc/rc.d/sshd restart"))
```

Comment:
Since both freebsd and openbsd have the same sshd restart command, this could probably be changed to:

```ruby
elsif host['platform'] =~ /(free|open)bsd/
           host.exec(Command.new("sudo /etc/rc.d/sshd restart"))
end
```